test/integration/targets/ec2_tag/tasks/main.yml

```diff
@@ -1,46 +1,44 @@
 ---
 # tasks file for test_ec2_tag
 - name: Set up AWS connection info
-  set_fact:
-    aws_connection_info: &aws_connection_info
+  module_defaults:
+    group/aws:
       aws_access_key: "{{ aws_access_key }}"
       aws_secret_key: "{{ aws_secret_key }}"
-      security_token: "{{ security_token }}"
+      security_token: "{{ security_token | default(omit) }}"
       region: "{{ aws_region }}"
-  no_log: true
-
-- block:
+  block:
     - name: Create an EC2 volume so we have something to tag
       ec2_vol:
         name: "{{ resource_prefix }} ec2_tag volume"
         volume_size: 1
         state: present
         zone: "{{ aws_region }}a"
-        <<: *aws_connection_info
       register: volume
 
-    - name: List the tags
-      ec2_tag:
```

Comment:
I would prefer to keep the test for the existing functionality, to make sure it doesn't get broken during the deprecation period.

Reply:
Fairpoint.  added it back